docs/docs/develop/build-client.mdx

```diff
@@ -77,7 +77,13 @@ You'll need an Anthropic API key from the [Anthropic Console](https://console.an
 Create a `.env` file to store it:
 
 ```bash
-echo "ANTHROPIC_API_KEY=<your key here>" > .env
+echo "ANTHROPIC_API_KEY=your_key_here" > .env
+```
+
+Make sure to replace `your_key_here` with your actual API key, **removing any angle brackets**. Your `.env` file should look like:
+
+```plaintext
+ANTHROPIC_API_KEY=sk-ant-api03-example1234567890abcdefghijklmno
```

Comment:
I think it's reasonable to remove the angle brackets if that causes confusion, but I feel like the other text does not add much.  (In particular, the part about "removing any angle brackets" no longer applies since we are removing the brackets from the example.)

``````suggestion
echo "ANTHROPIC_API_KEY=your-api-key-goes-here" > .env
``````